packages/config/config/devices/0x027a/zen26_1.0-1.0.json

```diff
@@ -12,14 +12,25 @@
 		}
 	],
 	"firmwareVersion": {
-		"min": "0.0",
-		"max": "255.255"
+		"min": "1.0",
+		"max": "1.0"
+	},
+	"associations": {
+		"1": {
+			"label": "Lifeline",
+			"maxNodes": 1,
```

Comment:
Please remove "supportsZWavePlus": true and the associations groups from all zwave plus devices unless you need a second isLifeline. We're in the process of cleaning up the old files.

Reply:
So I get your point with this (and I actually recently opened a bug because we aren't using our name anyway in zwave2jsmqtt), but currently I'm not including associations for zwave plus devices per our earlier conversations. Do you want me to add those back in? I'd assume the ones in the zwa files match the devices but maybe not and we may be able to clean the names up some as we go.

Reply:
Let's try without first. If we identify devices that need better labels, we can always add them back.

Reply:
I'll pull the associations sections out and clear the cache on my dev instance of zwavejs2mqtt later today and see what it comes back with.

Reply:
I have to go based on a Zen77 right now, because I haven't updated my Zen27 to the firmware that supports group 3 yet due to openzwave issues. The Zen77 reports groups 2 and 3 as "BASIC SET". According to Zooz support, group 3 should be SWITCH MULTILEVEL and send dimming events to associated devices.

Reply:
That seems pretty unhelpful and what we want to avoid. Basic Set of what?

Reply:
Okay, then leave the section in the config file :)

Reply:
What's odd, the Zen34 reports Group 3 as "SWITCH_MULTILEVEL CHANGE".